ArduCopter/config.h

```diff
@@ -327,6 +327,14 @@
 
 //////////////////////////////////////////////////////////////////////////////
 
+//////////////////////////////////////////////////////////////////////////////
+// Weathervane - allow vehicle to yaw into wind
+#ifndef WEATHERVANE_ENABLED
+# define WEATHERVANE_ENABLED !HAL_MINIMIZE_FEATURES
```

Comment:
I'm not sure `HAL_MINIMIZE_FEATURES` is right threshold for this, although I'm not really sure what would be better, we could do `BOARD_FLASH_SIZE > 1024`.